firefox-ios/Client/Frontend/Home/HomepageViewModel.swift

```diff
@@ -94,6 +94,7 @@ class HomepageViewModel: FeatureFlaggable, InjectedThemeUUIDIdentifiable {
     weak var delegate: HomepageViewModelDelegate?
     private var wallpaperManager: WallpaperManager
     private var logger: Logger
+    private let throttler = Throttler(seconds: 0.1)
```

Comment:
Minor nit: might be nice to make this property name slightly more descriptive (`viewWillAppearEventThrottler`?), the verbosity might be worth the added clarity. Not a big deal though.

Reply:
updated!